frontend/src/static/js/v_authorship.js

```diff
@@ -108,7 +114,7 @@ window.vAuthorship = {
       window.encodeHash();
     },
 
-    initiate() {
+    async initiate() {
```

Comment:
Actually, is it necessary to make this asynchronous here?

Reply:
No, but the alternative is `.then()` chaining is something I avoid for personal preference, as `loadAuthorship()` is an async function which I need to wait for.

Reply:
No, but the alternative is .then() chaining is something I avoid for personal preference, as loadAuthorship() is an async function which I need to wait for.